Nodejs/Product/Nodejs/Project/GlobalModulesNode.cs

```diff
@@ -1,101 +0,0 @@
-﻿//*********************************************************//
-//    Copyright (c) Microsoft. All rights reserved.
-//    
-//    Apache 2.0 License
-//    
-//    You may obtain a copy of the License at
-//    http://www.apache.org/licenses/LICENSE-2.0
-//    
-//    Unless required by applicable law or agreed to in writing, software 
-//    distributed under the License is distributed on an "AS IS" BASIS, 
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
-//    implied. See the License for the specific language governing 
-//    permissions and limitations under the License.
-//
-//*********************************************************//
-
-using System;
-using System.Linq;
-using Microsoft.NodejsTools.Npm;
-using Microsoft.VisualStudio;
-using Microsoft.VisualStudioTools.Project;
-
-namespace Microsoft.NodejsTools.Project {
-    internal class GlobalModulesNode : AbstractNpmNode {
```

Comment:
In addition to this, please remove usages of NpmBinCommand (which we use to locate the global modules directory) because that's been a source of completely un-reproducible perf issues and we've only ever been able to fix half the problem. 
![](https://cloud.githubusercontent.com/assets/654999/10720917/22e6107a-7bd7-11e5-9b54-0a8088bf79f8.png)